I add the validation security on the request to the methods

abdiel8real opened the pull request.

abdiel8real committed:
I add the validation security on the request to the methods

abdiel8real committed:
Merge pull request #2 from nathabdiel/feature/delete-registers

Feature/delete registers

abdiel8real committed:
I add the feature pagination to the component category

cesarureno commented on app/Http/Controllers/CategoriaController.php:
```diff
@@ -12,8 +12,9 @@ class CategoriaController extends Controller
     {
+        if (!$request->ajax()) return redirect('/');
         $categorias = Categoria::all();
         return $categorias;
```
Make this in only one line 
```suggestion
        return Categoria::all()
```

abdiel8real committed:
Fixed the 2 lines

Now there is no declaration statement now return in just one line in the categories

abdiel8real committed:
Merge branch 'master' into feature/http-security-petitions
